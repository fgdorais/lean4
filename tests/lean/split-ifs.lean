/-! Test that the `split` tactic doesn't use `Classical.em` to split ifs.
-/

theorem ifthisthenthat (n : Nat) : (if n = 0 then 0 else 0) = 0 := by split <;> rfl

#print axioms ifthisthenthat
