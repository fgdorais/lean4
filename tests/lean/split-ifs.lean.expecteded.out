'ifthisthenthat' does not depend on any axioms
